<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow_admin</class>
 <widget class="QMainWindow" name="MainWindow_admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>管理员用户</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 25 10pt &quot;微软雅黑 Light&quot;;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>651</width>
      <height>461</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QTabWidget" name="tabWidget_admin">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="usesScrollButtons">
        <bool>false</bool>
       </property>
       <widget class="QWidget" name="tab_course_o">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <attribute name="title">
         <string>必修课列表</string>
        </attribute>
        <widget class="QTableView" name="tableView_course_o">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>651</width>
           <height>431</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>105</number>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_course_e">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <attribute name="title">
         <string>选修课列表</string>
        </attribute>
        <widget class="QTableView" name="tableView_course_e">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>651</width>
           <height>431</height>
          </rect>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>105</number>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_student">
        <attribute name="title">
         <string>学生列表</string>
        </attribute>
        <widget class="QTableView" name="tableView_student">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>651</width>
           <height>431</height>
          </rect>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>105</number>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_teacher">
        <attribute name="title">
         <string>教师列表</string>
        </attribute>
        <widget class="QTableView" name="tableView_teacher">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>651</width>
           <height>431</height>
          </rect>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>105</number>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_admin">
        <attribute name="title">
         <string>管理员列表</string>
        </attribute>
        <widget class="QTableView" name="tableView_admin">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>651</width>
           <height>431</height>
          </rect>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>105</number>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 11pt &quot;微软雅黑 Light&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;管理员信息&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>141</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="3" column="2">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="0">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">font: 25 11pt &quot;微软雅黑 Light&quot;;</string>
       </property>
       <property name="text">
        <string>学院：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">font: 25 11pt &quot;微软雅黑 Light&quot;;</string>
       </property>
       <property name="text">
        <string>姓名：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">font: 25 11pt &quot;微软雅黑 Light&quot;;</string>
       </property>
       <property name="text">
        <string>用户名：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLabel" name="label_id">
       <property name="styleSheet">
        <string notr="true">font: 25 11pt &quot;微软雅黑 Light&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLabel" name="label_name">
       <property name="styleSheet">
        <string notr="true">font: 25 11pt &quot;微软雅黑 Light&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QLabel" name="label_institude">
       <property name="styleSheet">
        <string notr="true">font: 25 11pt &quot;微软雅黑 Light&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="add_user_btn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px;</string>
    </property>
    <property name="text">
     <string>添加用户</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_course_btn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>390</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px;</string>
    </property>
    <property name="text">
     <string>添加课程</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>111</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/image/photo.png);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_main">
    <property name="title">
     <string>菜单</string>
    </property>
    <addaction name="action_login"/>
    <addaction name="action_quit"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_course">
    <property name="title">
     <string>课程</string>
    </property>
    <addaction name="action_course_o"/>
    <addaction name="action_course_e"/>
    <addaction name="separator"/>
    <addaction name="action_course_add"/>
   </widget>
   <widget class="QMenu" name="menu_user">
    <property name="title">
     <string>用户</string>
    </property>
    <addaction name="action_student_d"/>
    <addaction name="action_teachet_d"/>
    <addaction name="action_admin_d"/>
    <addaction name="separator"/>
    <addaction name="action_user_add"/>
   </widget>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action_about"/>
    <addaction name="action_help"/>
   </widget>
   <widget class="QMenu" name="menu_function">
    <property name="title">
     <string>功能</string>
    </property>
    <addaction name="action_start_course"/>
   </widget>
   <addaction name="menu_main"/>
   <addaction name="menu_course"/>
   <addaction name="menu_user"/>
   <addaction name="menu_function"/>
   <addaction name="menu_help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_quit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="action_login">
   <property name="text">
    <string>重新登录</string>
   </property>
  </action>
  <action name="action_course_o">
   <property name="text">
    <string>查看必修课程</string>
   </property>
  </action>
  <action name="action_course_e">
   <property name="text">
    <string>查看选修课程</string>
   </property>
  </action>
  <action name="action_student_d">
   <property name="text">
    <string>查看学生用户</string>
   </property>
  </action>
  <action name="action_teachet_d">
   <property name="text">
    <string>查看教师用户</string>
   </property>
  </action>
  <action name="action_admin_d">
   <property name="text">
    <string>查看管理员用户</string>
   </property>
  </action>
  <action name="action_course_add">
   <property name="text">
    <string>添加课程</string>
   </property>
  </action>
  <action name="action_student_add">
   <property name="text">
    <string>添加学生用户</string>
   </property>
  </action>
  <action name="action_teacher_add">
   <property name="text">
    <string>添加教师用户</string>
   </property>
  </action>
  <action name="action_user_add">
   <property name="text">
    <string>添加用户</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="action_help">
   <property name="text">
    <string>帮助文档</string>
   </property>
  </action>
  <action name="action_start_course">
   <property name="text">
    <string>开放选课</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
